intelmq/tests/bots/experts/domaintools/test_expert.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+"""
+Testing GethostbynameExpertBot.
+"""
+
+import unittest
+
+import intelmq.lib.test as test
+from intelmq.bots.experts.domaintools.expert import DomaintoolsExpertBot
+
+EXAMPLE_INPUT = {"__type": "Event",
+                 "source.fqdn": "google.com",
+                 "time.observation": "2015-01-01T00:00:00+00:00"
+                 }
+EXAMPLE_OUTPUT = {"__type": "Event",
+                  "source.fqdn": "google.com",
+                  "extra": '{"domaintools_score": 0}',
+                  "time.observation": "2015-01-01T00:00:00+00:00"
+                  }
+NONEXISTING_INPUT = {"__type": "Event",
```

Comment:
Actually the lib raises the `BadRequestException` for this test which is not equivalent to 'not existing', but the result is similar.